src/ol/source/xyzsource.js

```diff
@@ -63,6 +69,16 @@ goog.inherits(ol.source.XYZ, ol.source.TileImage);
 
 
 /**
+ * Return the URLs used for this XYZ source.
```

Comment:
Add a note that this will be null when a `tileUrlFunction` was configured instead of `url` or `urls`.